scripts/test_mutation.py

```
@@ -1780,9 +1780,10 @@ def enqueue_three(queue_mode: str) -> tuple[list[str], list[str]]:
                 f"  cancel-in-progress: {cancel_rule}\n",
                 workflow,
             )
-        # Эфемерные GitHub-hosted VM не несут ни секретов, ни состояния,
-        # поэтому fork-PR — штатный режим опенсорс-гейта: workers не держат
-        # fork-гейтов, а каждая job закреплена на одной одноразовой метке.
+        # Эфемерные раннеры (GitHub-hosted или self-hosted без секретов)
+        # не несят ни секретов, ни состояния, поэтому fork-PR — штатный
+        # режим опенсорс-гейта: workers не держат fork-гейтов, а каждая
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `а` (CYRILLIC SMALL LETTER A). Did you mean `a` (LATIN SMALL LETTER A)?

(RUF003)